.changeset/slow-insects-cheat.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-techdocs': minor
+'@backstage/plugin-catalog': minor
+'@backstage/plugin-explore': minor
```

Comment:
Think this should be `patch` for the `explore` plugin as it's <1.0.0. But the rest are fine

Reply:
changed to patch for all of them as there the same changes